static/js/graphs_page.js

```diff
@@ -584,7 +594,7 @@ var graphPage = {
         graphPage.cyGraph.layout(layout);
 
     },
-    saveLayout: function (layoutName) {
+    saveLayout: function (layoutName, modalNameId) {
```

Comment:
I can rename `modalNameId` to something more intuitive. Any suggestions?

I need the ID of the modal element to toggle between the element's state only on successful callback. If I take it out will have to implement a callback (callback from a callback situation). I thought of making it modular, in case we use this function for other DOM elements.